### DISCUSSION TITLE
Welcome to lint Discussions!

### DISCUSSION THREAD

**Discussion by allanbowe** [sasjs/lint#70] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Masum-pixel** [comment]
Hi Team,
Thank you for updating few rules in SASjs Lint. Could you please add more rule for best practices of SAS Code ?
Please check this paper for your reference:

https://www.lexjansen.com/pharmasug/2022/AP/PharmaSUG-2022-AP-072.pdf
https://cpb-us-w2.wpmucdn.com/sites.wustl.edu/dist/8/1443/files/2015/10/Optimize-Your-Code.pdf

**Reply by allanbowe** [reply]
Hi @Masum-pixel - please try not to create duplicate posts!  We can use this one as the main version:  https://github.com/sasjs/lint/discussions/183
